web/src/app/admin/qualifications/qualifications.component.ts

```diff
@@ -12,11 +12,10 @@ export class QualificationsComponent implements OnInit {
   qualifications: QualificationViewModel[] = [] as QualificationViewModel[];
   isCreateOrUpdate  = true;
```

Comment:
what is the purpose of this variable ?

Reply:
its required for edit field

Reply:
remove it and find out some another way.